src/bots/liquidator.ts

```diff
@@ -1427,6 +1443,22 @@ tx: ${tx} `
 		borrowMarketIndextoLiq: number,
 		borrowAmountToLiq: BN
 	) {
+		const minAmount = this.minDepositToLiq[depositMarketIndextoLiq] || 0;
+
+		logger.debug(
+			`liqPerpPnl: Min liquidation for market ${depositMarketIndextoLiq} is ${minAmount}`
+		);
+		if (depositAmountToLiq < minAmount) {
```

Comment:
also it's probably better to do this min amount check before entering `liqPerpPnl`, so you cover all other possible liquidation modes as well. perhaps somewhere here: https://github.com/drift-labs/keeper-bots-v2/blob/ee823fdb9df8e3ffd3c022b90c811ba3ae3ffe40/src/bots/liquidator.ts#L1781

also might be a good idea to check `borrowAmountToLiq` too!

Reply:
I was also thinking that listing it in lamports (or base fractions) is not the most intuitive, so I may change that as well. Thanks.

Reply:
Re-reading the code, wouldn't `findBestSpotPosition` be the best place to filter these out, since that's where `depositAmountToLiq` comes from?

Reply:
> I was also thinking that listing it in lamports (or base fractions) is not the most intuitive, so I may change that as well (or at the very least document it clearly). Thanks.

yeah it's currently implied that perp amounts are in `BASE_PRECISION`, and spot amounts are in the respective spot asset precisions (i.e. lamports).


> Re-reading the code, wouldn't findBestSpotPosition be the best place to filter these out, since that's where depositAmountToLiq comes from?

yup that will be a good place to do the check as well